enterprise/frontend/src/embedding-sdk/components/private/PublicComponentStylesWrapper.tsx

```diff
@@ -30,7 +29,6 @@ export const PublicComponentStylesWrapper = styled.div`
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 
-  ${aceEditorStyles}
```

Comment:
These styles are now imported from the AceEditor component (and properly scoped there).